NEWPART.md

```diff
@@ -0,0 +1,203 @@
+
+# Quick guide to adding a new part to an existing family
+
+This documents what I did to add support for xc7a100t parts, 
+in hope that it is useful to someone adding another new part.
+
+_If you are **just** adding a new package for a device that is already handled, you can skip Steps 2..4!_
+
+
+### Step 0 ###
+Fork a copy of https://github.com/SymbiFlow/prjxray on GitHub (go to the page, click "Fork" button, select your own workspace).
+
+Clone your fork (replace "tcal-x" with your GitHub ID):
+
+```
+git clone git@github.com:tcal-x/prjxray.git
+cd prjxray
+```
+
+
+
+### Step 1 ###
+
+Follow the Project X-Ray setup instructions in the [README.md](https://github.com/SymbiFlow/prjxray/blob/master/README.md), up through Step 7, Option 1.  That is, source the `./download-latest-db.sh` script.   NOTE: if you might want to run tests with your own generated db, you'll need to take some extra steps here.   I sketch out what that process looks like at the end of this document.  
+
+
+
+### Step 2 ###
+
+Add a new settings file.  Usually you will start with an existing settings file and modify it.
+Assuming you're in prjxray/,
+
+```
+cp settings/artix7_200t.sh settings/artix7_100t.sh
```

Comment:
Please change to two instances of what needs to be changed.  One that makes clear the old and new, and an example.  So for example this block would be:

```
cp settings/<baseline settings>.sh settings/<new settings>.sh
git add settings/<new settings>.sh
```

Example:

```
cp settings/artix7_200t.sh settings/artix7_100t.sh
git add settings/artix7_100t.sh
```